include/boost/process/v2/detail/impl/process_handle_windows.ipp

```diff
@@ -113,6 +123,31 @@ void check_running_(HANDLE handle, error_code & ec, DWORD & exit_status)
         ec = detail::get_last_error();
 }
 
+#if !defined(BOOST_PROCESS_V2_DISABLE_UNDOCUMENTED_API)
+void suspend_(HANDLE handle, error_code & ec)
+{
+    auto nt_err = NtSuspendProcess(handle);
+    if (nt_err > 0xC0000000)
+       ec = detail::get_last_error();
+}
+
+void resume_(HANDLE handle, error_code & ec)
+{
+    auto nt_err = NtResumeProcess(handle);
+    if (nt_err > 0xC0000000)
+        ec = detail::get_last_error();
```

Comment:
We need to check if the GetLastError call is correct here. It's possble we need another error category, if the codes don't align.

Reply:
I'm not sure if GetLastError returns with these undocumented api's. Isn't that what the NTSTATUS is for?

Reply:
I think so - and we'd need an error_category for NTSTATUS then, which may end up horrific.

Reply:
yeah - i wouldn't overthink it, i would prefer being less paranoid for now and be satisfied with something less than perfect and fix it as needed if someone reports breakage or flaws, i as the only real way to know how to perfect it would be through hands on test cases imo